docs/zkEVM-custom-endpoints.md

```diff
@@ -0,0 +1,9 @@
+# zkEVM custom endpoints
```

Comment:
It would be nice to link this doc in the main README, so people actually find this easily